vagrant_variables.yml.linode

```diff
@@ -0,0 +1,51 @@
+---
+
+vagrant_box: 'linode'
+vagrant_box_url: 'https://github.com/displague/vagrant-linode/raw/master/box/linode.box'
+
+ssh_username: 'vagrant'
+ssh_private_key_path: '~/.ssh/id_rsa'
+
+cloud_distribution: 'CentOS 7'
+cloud_datacenter: 'newark'
+
+# Memory for each Linode instance, this determines price! See Linode plans.
+memory: 2048
+
+# The private network on Linode, you probably don't want to change this.
+subnet: 192.168.0
+
+# The interface the monitors should listen on, this is currently ignored for linode.
+eth: eth0
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 3
+osd_vms: 3
+mds_vms: 1
+rgw_vms: 0
+nfs_vms: 0
+rbd_mirror_vms: 0
+client_vms: 0
+iscsi_gw_vms: 0
+
+# Deploy RESTAPI on each of the Monitors
+restapi: true
+
+# INSTALL SOURCE OF CEPH
+# valid values are 'stable' and 'dev'
+ceph_install_source: 'dev'
+ceph_install_branch: 'master'
+
+# The sync directory changes based on vagrant box
+# Set to /home/vagrant/sync for Centos/7, /home/{ user }/vagrant for openstack and defaults to /vagrant
+# vagrant_sync_dir: /home/vagrant/sync
+
+os_tuning_params:
+  - { name: kernel.pid_max, value: 4194303 }
+  - { name: fs.file-max, value: 26234859 }
+
+ceph_conf_overrides:
```

Comment:
Hum `ceph_conf_overrides` can be used and should from `group_vars/all`, so I'd like to keep it that way. When it comes to optimizations, I have a local branch for "profiles". These use `ceph_conf_overrides` and provide some guidance on how to tune/tweak specific deployment. The fuse option falls down into this so it's a good candidate once I push the branch, until then I suggest we don't do anything.